src/lib/select/select.ts

```diff
@@ -420,7 +420,7 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
       .pipe(take(1), switchMap(() => this.optionSelectionChanges));
   });
 
-   /** Event emitted when the select has been opened. */
+   /** Event emitted when the select has been opened (true) or closed (false). */
```

Comment:
I think the true/false is implied from the `boolean` typing. Something like "Event emitter when the select panel has been toggled" is clearer.

Reply:
Ok, I'll change it, but... I tried to make it clear that true is attached to the opening action and false is attached to the closing action.